config/install/core.entity_form_display.paragraph.stanford_teaser_card.default.yml

```diff
@@ -0,0 +1,50 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.paragraph.stanford_teaser_card.field_p_teasercard_headline
+    - field.field.paragraph.stanford_teaser_card.field_p_teasercard_image
+    - field.field.paragraph.stanford_teaser_card.field_p_teasercard_link
+    - field.field.paragraph.stanford_teaser_card.field_p_teasercard_teaser
+    - image.style.thumbnail
```

Comment:
This image style will need to be pulled out as it is installed by default. 
Will help revise momentarily.